js/menu.js

```diff
@@ -1,30 +1,240 @@
+    const project = projects[item];
+
+    return `<div class="works-box" 
+    style="background: url(${project.img}) center center no-repeat; background-size: cover;"
+    id="${item}"
+    ${item === 'project0' ? '' : 'onclick="clickedDesktopBtn(this)"'}
+    >
+    <h2>${project.title}</h2>
+    <p>${project.description}</p>
+
+    <ul class="nav-buttons box-nav">
+      ${project.tech[0] === undefined ? '' : `<li><a href="">${project.tech[0]}</a></li>`}
+      ${project.tech[1] === undefined ? '' : `<li><a href="">${project.tech[1]}</a></li>`}
+      ${project.tech[2] === undefined ? '' : `<li><a href="">${project.tech[2]}</a></li>`}
+    </ul>
+
+    ${item === 'project0' ? '<img width=20 height=20 src="./imgs/socialIcons/Shape.png" alt="cursor">' : ''}
+
+    ${item === 'project0' ? '<button id="project0" onclick="clickedDesktopBtn(this)" class="btn-type-1 box-btn">See Project</button>' : ''}
+  </div> `;
+  });
+  return cards;
+}
+
+window.onload = () => {
+  document.querySelector('#works-box-container').innerHTML = mobileCards().join('');
+  document.querySelector('#works-box-container-desktop').innerHTML = desktopCards().join('');
+};
+
 function toggleHamburgerBtn() {
   const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const logo = document.querySelector('#header a');
+
+  // expression
   if (mobileMenu.style.display === 'none' || !mobileMenu.style.display) {
-    showMenu();
+    mobileMenu.style.display = 'block';
+    img.src = './imgs/x.svg';
+    logo.style.opacity = 0;
   } else {
-    hideMenu();
+    mobileMenu.style.display = 'none';
+    img.src = './imgs/IconMenu.svg';
+    logo.style.opacity = 1;
   }
 }
 
-toggleHamburgerBtn();
\ No newline at end of file
+toggleHamburgerBtn();
+
+// popup window
+function showPopup(project) {
+  // dynamic the tech
+  const lis = project.tech.map((tech) => `<li><a href="">${tech}</a></li>`);
+
+  // setup the popup
+  return `<div id="popup">
+  <div id="popup-content">
+    <div id="popup-title-X">
+      <h3>${project.title}</h3>
+      <button id="popup-X-btn" onclick="hidePopup()">
+        <img src="./imgs/x.svg" alt="close button">
+      </button>
+    </div>
+
+    <div id="popup-skill-btns">
+      <ul class="nav-buttons">
+        ${lis.join('')}
+      </ul>
+    </div>
+
+    <div id="popup-info">
+      <img width="100%" src="${project.img}" alt="project-image">
+
+      <div id="popup-info-details">
+        <p>${project.description}</p>
+
+        <div id="function-btns">
+          <button class="btn-type-1">
+            See Live
+            <img width="20" src="./imgs/socialIcons/popupIcon.svg" alt="popupIcon">
+          </button>
+          </button>
+          <button class="btn-type-1">
+            See Source
+            <img width="20" src="./imgs/socialIcons/gitIcon.svg" alt="gitIcon"></button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>`;
+}
```

Comment:
- [OPTIONAL ] Great job so far @khanh-devos Your project looks great 😉 However I noticed that the second popup is missing the title, the languages and the paragraph text. So kindly ensure that these elements appear on the popup as expected so as to provide users with a great user experience 😉 
- Your Design now
<img width="987" alt="Screen Shot 2023-05-18 at 2 22 18 PM" src="https://github.com/khanh-devos/MyPortfolio/assets/102699958/cab9492f-7abe-466c-bd81-255061dca465">

Reply:
yeah...